scripts/install_st2.sh

```diff
@@ -64,46 +84,18 @@ fi
 echo "*** Let's install some dev tools ***"
 
 if [[ -n "$RHTEST" ]]; then
-  if [[ "$RHMAJVER" == '6' ]]; then
-    sudo yum install -y centos-release-SCL
-    sudo yum install -y python27
-    echo "LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64:$LD_LIBRARY_PATH" | sudo tee -a /etc/environment
-    sudo ln -s /opt/rh/python27/root/usr/bin/python /usr/local/bin/python
-    sudo ln -s /opt/rh/python27/root/usr/bin/pip /usr/local/bin/pip
-    source /etc/environment
-    sudo yum install -y python-pip git
-  elif [[ "$RHMAJVER" == '7' ]]; then
-    sudo yum install -y epel-release
-    sudo yum install -y python
-    sudo yum install -y python-pip git
-  else
-    sudo yum install -y epel-release
-    sudo yum install -y python3
-    sudo yum install -y python3-pip git
-  fi
+  sudo yum install -y epel-release
+  sudo yum install -y python3
+  sudo yum install -y python3-pip git
 elif [[ -n "$DEBTEST" ]]; then
-  if [[ "$DEBCODENAME" == "xenial" ]]; then
-    sudo apt-get install -y python2.7 python-pip git
-  else
-    sudo apt-get install -y python3 python3-pip git
-  fi
+  sudo apt-get install -y python3 python3-pip git
```

Comment:
I think on one of the variants we had to use python3 rather than python3.6 as otherwise we ended up with an earlier python, trying to recall if it was U16 or CentOS 7.

Reply:
Ubuntu Xenial only has up to Python 3.5 available in the standard package repositories, I would bet it's that one. But I don't know how much this matters since it's just necessary to upgrade pip and virtualenv before the ST2 bootstrap script is run.

Reply:
```
    st2vagrant: *** Let's install some dev tools ***
    st2vagrant: Reading package lists...
    st2vagrant: Building dependency tree...
    st2vagrant: Reading state information...
    st2vagrant: E
    st2vagrant: :
    st2vagrant: Unable to locate package python36
    st2vagrant: E
    st2vagrant: :
    st2vagrant: Unable to locate package python36-pip
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Same story with `python3.6-pip`.

Reply:
I added a commit that has the provisioning script print out the exact version of Python, so we can be 100% sure from the logs what the default Python 3 minor version is.